protos/flyteidl/admin/project_domain_attributes.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+message ProjectDomainAttributes {
+    // Unique project id for which this set of attributes will be applied.
+    string project                 = 1;
+
+    // Unique domain id for which this set of attributes will be applied.
+    string domain                  = 2;
+
+    // Custom resource attributes which will be applied in cluster resource creation (e.g. quotas).
+    // Map keys are the names of variables in templatized resource files.
```

Comment:
case sensitive or not? maybe call that out here...

Reply:
good point, done